core/src/main/scala/scalaz/ApplicativeError.scala

```diff
@@ -0,0 +1,17 @@
+package scalaz
+
+trait ApplicativeError[F[_], S] extends Applicative[F]{
+
+  ////
```

Comment:
the point of the slashes is that you should modify `GenTypeClass` to generate this file (and `ApplicativeSyntax`) for you with a standard template; see #2004 for an example